save-deploy/README.md

```diff
@@ -183,19 +183,128 @@ Usually, not the whole stack is required for development. Application logic is p
    The resulting application settings may look like this: [screenshot](../info/img/github-oauth-app-settings.png).
 
 ## Local debugging
-You can run backend, orchestrator, preprocessor and frontend locally in IDE in debug mode.<br/>
-If you run on Windows, dependency `save-agent` is omitted because of problems with linking in cross-compilation.<br/>
-To run on Windows you need to compile `save-agent` on WSL and put `saveAgentDistroFilepath` to `%USERPROFILE%\.gradle\gradle.properties` <br/>
-For example:
+You can run backend, orchestrator, preprocessor and frontend locally in IDE in debug mode.
+
+#### Using `save-agent` executable on Windows
+
+If you run on Windows, dependency `save-agent` is omitted because of problems with linking in cross-compilation.
+To run on Windows, you need to build and package `save-agent` on WSL:
+
+```bash
+./gradlew :save-agent:copyAgentDistribution
+```
+
+and provide the path to the JAR archive which contains `save-agent.kexe` via the
+`saveAgentDistroFilepath` _Gradle_ property, by setting the above property
+either under project-specific `gradle.properties`, or, globally, under
+`%USERPROFILE%\.gradle\gradle.properties`, e.g.:
+
+```properties
+# gradle.properties
+saveAgentDistroFilepath=file:\\\\\\\\wsl$\\Ubuntu\\home\\username\\projects\\save-cloud\\save-agent\\build\\libs\\save-agent-0.3.0-alpha.0.48+1c1fd41-distribution.jar
```

Comment:
maybe refactor `gradle build` to support system environment to avoid changing `gradle.properties` ?

Reply:
This is already possible: you can set Gradle properties either globally, or on a per-project basis.

So, if you want your local Git repository clean, you can just putt al the necessary data under `~/.gradle/gradle.properties`.

Reply:
Additionally, you may set the property via the command line, e.g.:

```console
gradlew.bat gradlew.bat :save-backend:downloadSaveAgentDistro -PsaveAgentDistroFilepath=file:////wsl$/Debian/home/username/projects/save-cloud/save-agent/build/libs/save-agent-0.4.0-SNAPSHOT-distribution.jar
```

Reply:
Further addressed with b5c13d99c8a62ffb05728b53a0522285c5e6e55e